README.md

```diff
@@ -54,8 +54,33 @@ Then can run a dry-run upgrade to see the effect:
 helm upgrade --dry-run <release-name> <directory-containing-project-chart> --values <values-file>
 ```
 
-You can also compare the template yaml by running the following both before and after your changes, saving the output to files:
+You can also inspect the templated yaml by running the following:
 
 ```bash
 helm -n my-namespace template <release-name> <directory-containing-project-chart> --values <values-file>
 ```
+
+You can install the Helm diff plugin
+
+```bash
```

Comment:
Should helmdiff be a separate section?

Reply:
I'll add it to the top with the link to helm, and call it a perquisite